server/views/formPages/incident/report-may-already-exist.html

```diff
@@ -0,0 +1,96 @@
+{% extends "../../partials/layout.html" %} 
+{% from "govuk/components/button/macro.njk" import govukButton %} 
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% set pageTitle = pageData.pageTitle %} 
+
+
+{% block content %}
+<div class="govuk-body">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% if errors.length > 0 %}
+      {{
```

Comment:
inside a block should increase the level of indentation

Reply:
done